action.yaml

```diff
@@ -26,5 +21,11 @@ runs:
       # so we can use docker kill or docker stop. 
       sudo killall tracee-ebpf
       docker wait tracee-profiler
-
-      ${{ github.action_path }}/stop.sh ${{ inputs.workdir }} ${{ github.token }} ${{ inputs.fail-on-diff }} ${{ inputs.create-pr }}
+  - shell: bash
+    run: |
+      echo "Generating profile..."
+      ./profile/profile-gen /tmp/tracee/out/trace_$GITHUB_RUN_ID.jsonl > /tmp/tracee/profile.json
```

Comment:
The `> /tmp/tracee/profile.json` doesn't work, because once we map the volume in docker with `--privileged` this becomes a root folder, and the runner user doesn't have permission to create the file.